docs/pages/2019_MARVEL_Psik_MaX/sections/setup.rst

```diff
@@ -20,53 +20,75 @@ The software on this VM is based on `Quantum Mobile
-   the work of your colleague.
+   If you decide to work in pairs, one of you should forward their
+   credentials to the other and you should both use the same IP address and
+   ssh key. Since you will be sharing the VM and user account, be careful not
+   to delete the work of your colleague.
 
 Linux and MacOS
 ~~~~~~~~~~~~~~~
 
+It's recommended for you to place the ssh key you received in a folder
+dedicated to your ssh configuration, to do so:
+
 -  If not already present, create a ``.ssh`` directory in your home
    (``mkdir ~/.ssh``), and set its permissions: ``chmod 700 ~/.ssh``
 
--  Copy the two keys ``aiida_tutorial_NUM`` and ``aiida_tutorial_NUM.pub`` 
-   in the ``~/.ssh`` directory 
+-  Copy the two keys ``aiida_tutorial_NUM`` and ``aiida_tutorial_NUM.pub``
+   in the ``~/.ssh`` directory
 
 -  Set the correct permissions on the private key:
-   ``chmod 600 ~/.ssh/aiida_tutorial_NUM``.
-   You can check check with ``ls -l`` that the permissions of this file are now ``-rw-------``.
+   ``chmod 600 ~/.ssh/aiida_tutorial_NUM``. You can check check with ``ls -l``
+   that the permissions of this file are now ``-rw-------``.
+
+After that ssh key is in place, you can go two routes, the first through an
+ssh configuration file specific to the IP address you are going to connect, or,
+using a command in your shell with several configuration options.
 
--  Create (or modify) the ``~/.ssh/config`` file, adding the following lines:
+If you prefer the configuration route, first you want to add the following code
+block to your ``~/.ssh/condfig`` file,
 
-   .. code:: console
+.. code::
 
-       Host aiidatutorial
-         Hostname IP_ADDRESS
-         User aiida
-         IdentityFile ~/.ssh/aiida_tutorial_NUM
-         ForwardX11 yes
-         ForwardX11Trusted yes
-         LocalForward 8888 localhost:8888
+     Host aiidatutorial
+          Hostname 34.242.168.15
+          User max
+          IdentityFile ~/.ssh/aiida-tutorial-max.pem
+          ForwardX11 yes
+          ForwardX11Trusted yes
+          LocalForward 8888 localhost:8888
 
-   where you replace ``IP_ADDRESS`` with the IP address provided to you.
+Remember to update the ``Hostname`` IP address, for the one you received, after
+adding this configuration you can connect to the server using,
+
+.. code:: console
 
--  You should now be able to ``ssh`` to your virtual machine using simply
+     ssh aiidatutorial
 
-   .. code:: console
+All the options (user, x11 forwarding, etc) will be implied from this
+configuration file using that simple command.
+
+If you instead prefer to use a copy-paste ready command, you can skip the
+configuration block and instead use the following command straight into your
+console:
+
+.. code:: console
 
-         ssh -X -C aiidatutorial
+     ssh \
+          -i ~/.ssh/aiida-tutorial-max.pem \
+          -L 8888:localhost:8888 \
+          -X -C \
+          max@34.242.168.15
```

Comment:
use same placeholders and make it one line so that it is easier to copy

Reply:
It can be copy/pasted as is in the terminal, even with select then middle click.